src/us/kbase/workspace/test/kbase/JSONRPCLayerTest.java

```diff
@@ -783,6 +803,36 @@ public void saveProvenance() throws Exception {
 				new HashMap<String, String>(),
 				new HashMap<StringEpoch, StringEpoch>());
 	}
+	@Test
+	public void saveProvWithBadTime() throws Exception {
```

Comment:
Moved from the JSONRPCTester class and did a bit of cleanup since there's only one case now